src/components/Script.jsx

```diff
@@ -19,44 +19,30 @@ const Script = ({ actors, currentScript }) => {
   // create div and p elements for each chunk of lines
   const lineChunks = [];
   const characterSet = new Set(currentCharacters);
-  let currentActorCharacter;
 
   for (let i = 0; i < script.length; i++) {
     // checks if the character name for this chunk is the name of a character assigned to the current actor
     // remove the dot because some scripts have a dot after the name of the character, ie VIOLA.
     const name = script[i][0].replace('.', '');
     if (characterSet.has(name)) {
-      currentActorCharacter = `currentActor`;
+      lineChunks.push(<pre className='currentActor'>{script[i].join('\n')}</pre>);
```

Comment:
I replaced <blockquote> with <pre> so we wouldn't need <br> since <pre> renders newlines instead of skipping them like other html elements.  
I also only add a class when the actor has a line since we can use the presence of that class to tell the different line chunks apart rather than needing a class for each.